external/radiation/python/getdata.py

```diff
@@ -0,0 +1,240 @@
+import numpy as np
```

Comment:
Since this is a new file...it would be nice to add it to our linting rules. These are set here: https://github.com/ai2cm/fv3net/blob/279bdb782fa3232e2cc792f7bb97fcf86c050475/.pre-commit-config.yaml#L18

Reply:
I will allow lint/reformat in a next PR